### DISCUSSION TITLE
Toggle flashlight

### DISCUSSION THREAD

**Discussion by MsaterSgnt** [feelfreelinux/octo4a#155] [Q&A]
Is there a way to keep the flashlight enabled while not in the control tab viewing the camera?

**Comment by Grover49** [comment]
This is certainly not an answer and I would not claim to  know what I am talking about. But it would seem to me if the settings screen has a box to enable or disable the light there should be a script we could run to do the same thing, and then with a plugin to issue custom terminal commands we should be able to add a "light" switch.
